pandas/core/series.py

```diff
@@ -2939,6 +2943,22 @@ def sort_values(
         2    d
         0    z
         dtype: object
+
+        >>> s = pd.Series(['a', 'B', 'c', 'D', 'e'])
+        >>> s.sort_values()
+        1    B
+        3    D
+        0    a
+        2    c
+        4    e
+        dtype: object
+        >>> s.sort_values(key=str.lower)
```

Comment:
blank line, add a line of expl what you are doing

Reply:
updated.